### PR TITLE
 Node == 🐢 |  Elixir == 🚀 

### PR DESCRIPTION
**Pull request by nelsonic**
Hi @samhstn!

In the interest of "_fairness_", I've updated the Elixir code to use a built-in **`Enum.uniq`** function.
This allows Elixir (BEAM) to parallelize the computation across all CPU cores
which is _always_ be faster than Node.js.
(_on my "slow" 4-core Mac Elixir is 10x faster_ ... see: https://github.com/samhstn/array-iteration-benchmarks/issues/1#issuecomment-457746357 )

I understand that you were trying to _manually_ iterate over the Array of JSON strings,
but I don't feel this is a "_fair_" comparison ... There is an appropriate "tool" for this job,
there's no point hand-crafting a slower algorithm (`reduce` |> `Enum.find_index`...) 
if there's already an _optimised_ way of dealing with this "problem".

### CODE REVIEW

**Comment by nelsonic** [comment]
@samhstn Hope your morning is going well. (_are you already off to Cabo Verde?_ 🤔)
Please let me know if you prefer me _not_ to alter the Elixir code 🐎 
and limit the PR to fixing the `mix deps.get` and Markdown issues. 📝 
Stoked that you're being _proactive_ about benchmarking perf. 😍

File: README.md [hunk 1]
```diff
@@ -15,17 +15,21 @@ Languages being tested:
 + Node
 + Elixir
 
-### Findings
+### Findings
 
 Test your own findings by running:
 
 ```bash
+mix deps.get
```
Comment (samhstn) [inline README.md]: 👍 nice catch

File: lib/bench.ex [hunk 2]
```diff
@@ -2,20 +2,11 @@ defmodule Mix.Tasks.Bench do
...
-    strings
-    |> Enum.with_index()
-    |> Enum.reduce(0, fn {str, i}, count ->
-      if Enum.find_index(strings, &(&1 == str)) == i do
-        count
-      else
-        count + 1
-      end
-    end)
+    length(strings) - length(Enum.uniq(strings))
```
Comment (samhstn) [inline lib/bench.ex:5]: So much nicer 😍

File: README.md (hunk 1 above)
Comment (samhstn) [reply README.md]: Actually could we make this: `mix do deps.get, compile`

(makes for a nicer output when running for first time)

**Review by samhstn — COMMENTED** [review]
Definitely a much nicer solution for `count_duplicates` function 🚀 

While we're here, I just ran `mix deps.get && ./run.sh`. Adding `mix compile` in the middle makes for a nicer output (let me know your thoughts? - not too fussed though, happy to merge either way).

Looking forward to getting this merged, then looking at some other operations which I found were quite slow in Elixir!

File: README.md (hunk 1 above)
Comment (nelsonic) [reply README.md]: Good point. 👨🏻‍💻

Comment (nelsonic) [reply README.md]: Updated. :shipit:

File: lib/bench.ex (hunk 2 above)
Comment (nelsonic) [reply lib/bench.ex:5]: I couldn't idly stand by and see **`Elixir`** be (_artificially_) "slow" 🐌 
when there is a _much faster_ and more elegant way of achieving the same result 🐎 
using set theory and a built-in function. 😉 

Looking forward to the _next_ "Kata". 

# 🥋

**Review by samhstn — APPROVED** [review]
🚀

**Pull request merged**